src/moin/scripts/migration/moin19/import19.py

```diff
@@ -245,8 +245,8 @@ def __init__(self, backend, path, itemname):
     def iter_revisions(self):
         revisionspath = os.path.join(self.path, 'revisions')
         try:
-            # rather use this or a range(1, self.current+1)?
-            fnames = os.listdir(revisionspath)
+            # alternative method is to generate file names using range(1, self.current+1)
+            fnames = sorted(os.listdir(revisionspath))
```

Comment:
good that "back then", i formatted the rev file names with leading zeros, so sorted() now just works. :-)